dll/win32/browseui/internettoolbar.cpp

```diff
@@ -607,8 +607,35 @@ HRESULT STDMETHODCALLTYPE CMenuCallback::CallbackSM(LPSMDATA psmd, UINT uMsg, WP
 
 CInternetToolbar::CInternetToolbar()
 {
+    HKEY hKey;
+
+    // Set the toolbar to locked by default if the registry key is missing.
+    fLocked = true;
+    if (RegOpenKeyExW(HKEY_CURRENT_USER,
```

Comment:
Further down the file the function `SHRegGetBoolUSValueW` is used, any particular reason why you did not use that here?